lms/djangoapps/instructor_task/tests/test_api.py

```diff
@@ -220,6 +221,15 @@ def test_submit_ora2_request_task(self):
 
             mock_submit_task.assert_called_once_with(request, 'ora2_responses', get_ora2_responses, self.course.id, {}, '')
 
+    def test_submit_ora2_request_task_email(self):
```

Comment:
similar to the comment below we could rename this to 

test_submit_ora2_email_request_task